src/calculator-footer.tsx

```diff
@@ -0,0 +1,32 @@
+import { msg } from '@lit/localize';
+
+const toNbsp = (s: string) => s.replace(' ', '\u00a0');
+
+export const CalculatorFooter = () => (
+  <slot name="footer">
```

Comment:
Does the `slot` stuff still work in React-rendered context? I'm assuming so because it's a browser engine concern not a Lit concern?

Reply:
Yeah, the Shoelace components use slotting extensively and it's not been a problem.